Source/CarthageKit/Dependency.swift

```diff
@@ -1,16 +1,21 @@
 import Foundation
 import Result
+import ReactiveSwift
+import ReactiveTask
 import Tentacle
 
 /// Uniquely identifies a project that can be used as a dependency.
 public enum Dependency {
 	/// A repository hosted on GitHub.com or GitHub Enterprise.
+	/// This is treated as case-insensitive.
 	case gitHub(Server, Repository)
```

Comment:
Both `Server` and `Repository` are case-insensitive for equality:

- https://github.com/mdiep/Tentacle/blob/90c57572d15c56d591e671bbe4fc37f7288df060/Sources/Tentacle/Server.swift#L54-L65
- https://github.com/mdiep/Tentacle/blob/90c57572d15c56d591e671bbe4fc37f7288df060/Sources/Tentacle/Repository.swift#L28-L31

Reply:
https://github.com/Carthage/Carthage/pull/2185/files#diff-d36e2d0823f4b902aeb9274469b117c0R120